app/components/widgets/forms/image-upload.js

```diff
@@ -8,6 +8,7 @@ export default Component.extend({
   selectedImage   : null,
   allowDragDrop   : true,
   requiresDivider : false,
+  speakerimg      : false,
```

Comment:
No need

Reply:
@kushthedude , it is needed , added in imageupload to check if it is in speaker form or not. 
Other wise have to write a whole computed property.